src/main/java/model/RacingGame.java

```diff
@@ -0,0 +1,40 @@
+package model;
+
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+public class RacingGame {
+
+    private static final int RANDOM_BOUND = 10; // 난수 범위 0-9
+    private final List<Car> cars;
+    private final Random random = new Random();
+
+    public RacingGame(List<Car> carNames) {
+        this.cars = carNames;
+    }
+
+    // 레이스 진행
+    public void race() {
+        for (Car car : cars) {
+            int randomNum = random.nextInt(RANDOM_BOUND);
+            car.move(randomNum);
+        }
+    }
+
+    public List<Car> getCars() {
+        return cars;
+    }

```

Comment:
```suggestion
    public List<Car> getCars() {
        return List.copyOf(cars);
    }
```

클래스의 멤버 변수로 있는 컬렉션을 그대로 반환하게 되면 무슨 일이 일어날까요? 🤔

Reply:
외부 코드에서 컬렉션을 마음대로 수정할 수 있게 되어 불변성을 보장하지 못한다는 단점이 있습니다..
말씀해주신대로 `List.copyOf`를 사용해서 불변 리스트를 반환하게 리펙토링 시켰습니다 감사합니다 !